apps/backend/src/api/vms.py

```
@@ -6,14 +6,17 @@
 """
 
 import logging
+from typing import Any, Dict
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)